iis-common/src/test/resources/integration-test-default.properties

```diff
@@ -29,3 +29,5 @@ oozie.execution.log.file.location=target/extract-and-run-on-remote-host.log
 
 # location of maven executable
 maven.executable=mvn
+
+oozieLauncherQueueName=default_oozie_launcher
```

Comment:
Those are default properties so we don't need to provide it explicitly e.g. on jenkins worker node, right?